docs/Step_10_Property_Formatting_Using_Data_Types_9252ee4.md

```diff
@@ -52,7 +52,7 @@ sap.ui.require([
 				zip: "69190",
 				country: "Germany"
 			}*HIGHLIGHT START*,
-			"salesToDate" : 12345.6789,
+			"salesAmount" : 12345.6789,
```

Comment:
Remove quotation marks around salesAmount, they are not needed. Sames is true for currencyCode